airflow/providers/amazon/aws/hooks/s3.py

```diff
@@ -1369,10 +1362,6 @@ def download_file(
         """
         Download a file from the S3 location to the local file system.
 
-        Note:
-            This function shadows the 'download_file' method of S3 API, but it is not the same.
-            If you want to use the original method from S3 API, please use 'S3Hook.get_conn().download_file()'
-
```

Comment:
Seems like you do not rebase on current main, this changes made by previous PR

Reply:
Strange..just rebased it again.

Reply:
Just return in back. 

Or if you use PyCharm: `Right Click On file -> Git -> Compare with branch ... -> main` or `Right Click On file -> Git -> Compare with revision ... -> select revision which before you changes` and manually revert changes which you do not make on this PR. Same for https://github.com/apache/airflow/pull/37886#discussion_r1511795036

It is the easiest way that I know in case if incidentally resolve conflicts

![image](https://github.com/apache/airflow/assets/3998685/1ae29e5c-5c9e-40ab-ba46-c1f191aaa231)